### DISCUSSION TITLE
about-the-author/csdn-1000wan

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#267] [Announcements]
# about-the-author/csdn-1000wan

大家好，我是二哥呀！ 离上一个一千万 (https://mp.weixin.qq.com/s/8AylY_3EA5wWaKKU-qvDYw)，还是上一次。 我努力的回想着，回想自己在 2021 年做出了哪些耀眼的成绩，正襟危坐，回想良久，也只想到这最后一件：CSDN 的博文访问量也突破了一千万。 但这算不算得上是成绩，很难说，因为喜欢这个平台的人有很多...

https://javabetter.cn/about-the-author/csdn-1000wan.html

**Comment by hd5723** [comment]
现在2024了，祝福下。

**Comment by Oranzh** [comment]
现在2025了，祝福下。

**Comment by Oso-Kuan** [comment]
It's 2026 now, best wishes.
